packages/database/src/migrations/20210628230622-LESMISAgeOfGrade1Entrance-modifies-data.js

```diff
@@ -0,0 +1,229 @@
+      '...': '*',
+    },
+    {
+      transform: 'select',
+      "'sort_order'":
+        "translate($row.name, { 'Age 4': 1, 'Age 5': 2, 'Age 6': 3, 'Age 7': 4, 'Age 8': 5, 'Age 9': 6, 'Age 10': 7, 'Age 11': 8, 'Age 12': 9, 'Age 13': 10, 'Age 14': 11, 'Age 15': 12 })",
+      '...': '*',
+    },
+    {
+      transform: 'sort',
+      by: '$row.sort_order',
+    },
+    {
+      transform: 'select',
+      "'Age 4_metadata'":
+        "eq($row.name, 'Age 4') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 5_metadata'":
+        "eq($row.name, 'Age 5') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 6_metadata'":
+        "eq($row.name, 'Age 6') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 7_metadata'":
+        "eq($row.name, 'Age 7') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 8_metadata'":
+        "eq($row.name, 'Age 8') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 9_metadata'":
+        "eq($row.name, 'Age 9') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 10_metadata'":
+        "eq($row.name, 'Age 10') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 11_metadata'":
+        "eq($row.name, 'Age 11') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 12_metadata'":
+        "eq($row.name, 'Age 12') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 13_metadata'":
+        "eq($row.name, 'Age 13') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 14_metadata'":
+        "eq($row.name, 'Age 14') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'Age 15_metadata'":
+        "eq($row.name, 'Age 15') ? { numerator: $row.Numerator, denominator: $row.Denominator } : undefined",
+      "'value'": '$row.Numerator/$row.Denominator',
+      '...': ['name'],
+    },
+  ],
+};
+
+const FRONT_END_CONFIG = {
+  name: 'Age of Grade 1 Entrance',
+  type: 'chart',
+  chartType: 'pie',
+  periodGranularity: 'one_year_at_a_time',
+  valueType: 'fractionAndPercentage',
+  presentationOptions: {
+    'Age 4': { color: '#4caf50' },
+    'Age 5': { color: '#8bc34a' },
+    'Age 6': { color: '#acd042' },
+    'Age 7': { color: '#cddc39' },
+    'Age 8': { color: '#ffeb3b' },
+    'Age 9': { color: '#ffc107' },
+    'Age 10': { color: '#ffad04' },
+    'Age 11': { color: '#ff9800' },
+    'Age 12': { color: '#ff7811' },
+    'Age 13': { color: '#ff5722' },
+    'Age 14': { color: '#f44336' },
+    'Age 15': { color: '#aa2e25' },
```

Comment:
These are mostly just material ui 500 colours from `green > yellow > orange > red`, but I needed more colours so a few are just the midpoint between their neighbours